cmd/restart.go

```diff
@@ -39,9 +55,56 @@ var restartJobCmd = &cobra.Command{
 	Run:   restartJob,
 }
 
+var restartTasksCmd = &cobra.Command{
+	Use:   "tasks",
+	Short: "Restart tasks for a Job.",
+	Run:   restartTasks,
+}
+
 func restartJob(cmd *cobra.Command, args []string) {
 	key := aurora.JobKey{Environment: *env, Role: *role, Name: *name}
 	if err := client.RestartJob(key); err != nil {
 		log.Fatal("unable to create Aurora job: ", err)
 	}
 }
+
+func restartTasks(cmd *cobra.Command, args []string) {
+	log.Infof("Restarts task [Env:%s Role:%s Name:%s Instance:%s Monitor:%s]\n", *env, *role, *name, *instances, strconv.FormatBool(monitor))
+
+	//Set jobKey for the tasks to be killed.
+	task := realis.NewTask().
+		Environment(*env).
+		Role(*role).
+		Name(*name)
+
+	/*
+	* In the following block, we convert instance numbers, which were passed as strings, to integer values
+	* After converting them to integers, we add them to a slice of type int32.
+	 */
+
+	splitString := strings.Split(*instances, ",")
+	instanceList := make([]int32, len(splitString))
+
+	for i := range instanceList {
+		splitString[i] = strings.TrimSpace(splitString[i])
+		var instanceNumber int
+		var err error
+		if instanceNumber, err = strconv.Atoi(splitString[i]); err != nil {
```

Comment:
line 90&91 are not needed.
you can do
if instanceNumber, err :=;

Reply:
That won't work, since instanceNumber in line 96 would be undefined.
Line 96: instanceList[i] = int32(instanceNumber)